poutyne/framework/model.py

```diff
@@ -1007,7 +1007,7 @@ def _transfer_loss_and_metrics_modules_to_right_device(self):
         if isinstance(self.loss_function, torch.nn.Module):
             self.loss_function.to(self.device)
 
-        for metric in self.batch_metrics:
+        for metric in self.batch_metrics + self.epoch_metrics:
             if isinstance(metric, torch.nn.Module):
```

Comment:
Also this approach will allow to compute precision, recal and F1 score only with the FBeta right ? (What we previously spoke)

Reply:
I don't think that changes anything related to that.

Reply:
Ok.